molo/core/models.py

```diff
@@ -45,6 +79,29 @@ class Main(Page):
     parent_page_types = []
     subpage_types = ['core.LanguagePage']
 
+    COMMENTING_STATE_CHOICES = (
+        ('O', 'Open'),
+        ('C', 'Closed'),
+        ('D', 'Disabled'),
+        ('T', 'Timestamped'),
+    )
+    commenting_state = models.CharField(max_length=1,
+                                        choices=COMMENTING_STATE_CHOICES,
+                                        blank=False,
```

Comment:
Yes, it's not allowed to be blank because this effectively defines the default settings for the whole site. Anything that doesn't have settings set will eventually fall back to checking these ones.